cmd/server/app/server.go

```diff
@@ -350,7 +354,7 @@ func (p *Proxy) runAdminServer(o *options.ProxyRunOptions, server *server.ProxyS
 		}
 	}
 	adminServer := &http.Server{
-		Addr:           net.JoinHostPort(o.AdminBindAddress, strconv.Itoa(o.AdminPort)),
+		Addr:           fmt.Sprintf("127.0.0.1:%d", o.AdminPort),
```

Comment:
Why was this changed? Looks like maybe a bad rebase?

Reply:
+1. However this was merged Sept 2, over 3 months ago.